lib/config/configuration.go

```diff
@@ -1208,17 +1209,44 @@ func applyWindowsDesktopConfig(fc *FileConfig, cfg *service.Config) error {
 	}
 	ldapPassword, err := os.ReadFile(fc.WindowsDesktop.LDAP.PasswordFile)
 	if err != nil {
-		return trace.WrapWithMessage(err, "loading LDAP password from file %v",
+		return trace.WrapWithMessage(err, "error loading LDAP password from file %v",
 			fc.WindowsDesktop.LDAP.PasswordFile)
 	}
+
+	// If a CA file is provided but InsecureSkipVerifyCA is also set to true, throw an
+	// error to make sure the user isn't making a critical security mistake (i.e. thinking
+	// that their LDAPS connection is being verified to be with the CA provided, but it isn't
+	// due to InsecureSkipVerifyCA == true ).
+	if fc.WindowsDesktop.LDAP.DerEncodedCAFile != "" && fc.WindowsDesktop.LDAP.InsecureSkipVerifyCA {
+		return trace.Wrap(trace.BadParameter(
```

Comment:
You sure this should be an error and not just a warning we log to the console?

Reply:
I'm just going with "better safe than sorry", I'm open to the idea that I'm being paranoid.